lib/jsdom/living/svg/basic-types.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
```

Comment:
Should this file go in a helpers/ subdirectory or similar?

Reply:
Sure.